blocks/tabs-cards/tabs-cards.css

```diff
@@ -36,21 +38,48 @@
       }
     }
   }
+  
+  & .tab-component {
+    p:last-child {
+        display: block;
+    }
+  }
 
-  .esri-event-card, .other-event-card {
+  .event-card {
     padding: 0.5rem;
     flex: 0 0 100%;
     box-sizing: border-box; 
+    animation-delay: 0.05s;
+    animation-duration: 0.55s;
+    animation-fill-mode: both;
+    animation-name: in-up;
+    animation-timing-function: ease-in-out;
```

Comment:
Animations exist in the Calcite components. The class `.calcite-animate` sets up animations; `.calcite-animate_in-up` should replicate what you have here.